src/Content/Text/Markdown.php

```diff
@@ -34,6 +34,7 @@ public static function convert($text, $hardwrap = true) {
 		$MarkdownParser->hard_wrap = $hardwrap;
 		$MarkdownParser->code_class_prefix = 'language-';
 		$html = $MarkdownParser->transform($text);
+		$html = str_replace('<a href="#', '<a href="' . ltrim($_SERVER['REQUEST_URI'], '/') . '#', $html);
```

Comment:
Are you sure there aren't any `<a>` tag with attributes before the `href`?

Reply:
No, I am not sure. You are right, I will change it to a preg_replace.

Reply:
Or maybe just `str_replace('href="#', 'href="' . ltrim($_SERVER['REQUEST_URI'], '/') . '#', $html)`

Reply:
hmm .. can we be sure, that this is always part of a html <a element ?

BTW .. I just found https://regex101.com/ which also includes a regex debugger. The debugger shows very good how regex works. nice.